packages/commerce/src/hooks/use-commerce-machine.tsx

```diff
@@ -8,38 +8,44 @@ type CreateCommerceMachineProps = {
   sellable: SellableResource
   upgradeFromSellable?: SellableResource
   stripePriceId?: string
+  pricingApiUrl?: string
 }
 
 const createCommerceMachine = ({
   sellable,
   upgradeFromSellable,
   stripePriceId,
+  pricingApiUrl = `${process.env.NEXT_PUBLIC_AUTH_DOMAIN}/api/v1/sellable_purchases`,
```

Comment:
without material changes to this machine, we can send an overridable pricingApiUrl in so that when prices are loaded it's from the correct spot